doc/developers/tips.rst

```diff
@@ -163,6 +163,12 @@ Issue/Comment: Linking to comments
 
         Please use links to comments, which make it a lot easier to see what you are referring to, rather than just linking to the issue. See [this](https://stackoverflow.com/questions/25163598/how-do-i-reference-a-specific-issue-comment-on-github) for more details.
 
+Issue: Usage question
+    ::
+
+        Please, if any usage question comes up, it would be easier for you to make a search in `Stack Overflow`_ with the [scikit-learn] and [python] tags. Your question might have already been answered and so there will be no need of waiting a response of an author or a developer. On the other hand, if you have also 
```

Comment:
We would use this reply in the context that someone has posted an issue that *is* (or appears to be) a usage question. So "if any usage question comes up" is not an appropriate response.